Framework/Kernel/test/MersenneTwisterTest.h

```diff
@@ -139,10 +139,10 @@ class MersenneTwisterTest : public CxxTest::TestSuite {
 
 private:
   void assertSequenceCorrectForSeed_39857239(MersenneTwister &randGen) {
-    double expectedValues[10] = {
-        0.203374452656, 0.597970068222, 0.120683325687,  0.92372657801,
-        0.734524340136, 0.467380537419, 0.0712658402044, 0.204503614921,
-        0.487210249063, 0.885743656661};
+    double expectedValues[10] = {0.597970068269, 0.923726578038, 0.46738053759,
```

Comment:
While I'm not certain of this, I believe that C++11 implementation of Mersenne Twister algorithm is different to that of the boost version (as Mersenne Twister is a class of RNG's rather than a specific type, so can produce different numbers for the same seed).